public/js/index.js

```diff
@@ -1408,7 +1408,11 @@ $(document).ready(function () {
     // Helpers.
     $('.delete-button').click(function () {
         var $this = $(this);
-        $('.delete.modal').modal({
+        var filter = "";
+        if ($this.attr("id")) {
+          filter += "#"+$this.attr("id")
+        }
+        $('.delete.modal'+filter).modal({
```

Comment:
I have check that all .delete-button don't have id so that this change don't interact with it. I will double check and confirm it.